reana_commons/operational_options.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""REANA Workflow Operational Options implementation utils."""
+
+import copy
+import subprocess
+import sys
+
+import click
+
+from reana_commons.errors import REANAValidationError
+
+available_options = {
+    'CACHE': {'serial': True},
+    'FROM': {'serial': True},
+    'TARGET': {'serial': True, 'cwl': lambda: '--target'},
+    'toplevel': {'yadage': True},
+    'initdir': {'yadage': True}
+}
```

Comment:
Wouldn't we achieve the same if we change the dict to look simpler:
```suggestion
available_options = {
    'CACHE': {'serial': 'CACHE'},
    'FROM': {'serial': 'FROM'},
    'TARGET': {'serial': 'TARGET', 'cwl': '--target'},
    'toplevel': {'yadage': 'toplevel'},
    'initdir': {'yadage': 'initdir'}
}
```

And then inside `validate_operational_options` we simplify the implementation, you would still get `None` if there is no translation to a given engine, and in the case it exists it will be a string, which is fine `True`.